jobQueue/setup.js

```diff
@@ -18,19 +18,47 @@ const conf = require('../config');
 const featureToggles = require('feature-toggles');
 const urlParser = require('url');
 const kue = require('kue');
+const activityLogUtil = require('../utils/activityLog');
+
 const redisOptions = {
   redis: conf.redis.instanceUrl.queue,
 };
+
+const express = require('express');
+const server = require('http').Server(express());
+const SHUTDOWN_TIMEOUT = 5000;
```

Comment:
why hard-coded? shouldn't it be config with default value and env var overrate?

Reply:
Agree! 12 factors always :)